src/utils/constants.ts

```diff
@@ -11,6 +11,26 @@ export const WS_ONLY_METHODS = [
   'newBlockFilter',
 ]
 
+export const EVM_CHAINS = [
```

Comment:
Maybe AVAX should be in there as it has C-chain?

But more importantly, I think this list will need to be shared with the UI so that it can expose this as a feature. So, DB?

Reply:
> Maybe AVAX should be in there as it has C-chain?
> 
> But more importantly, I think this list will need to be shared with the UI so that it can expose this as a feature. So, DB?

Definitely. I think we could add a boolean called `evm` to the blockchain collection.